lib/Tooling/Refactor/ImplementDeclaredMethods.cpp

```diff
@@ -408,21 +426,15 @@ ImplementDeclaredObjCMethodsOperation::runInImplementationAST(
   std::string MethodString;
   llvm::raw_string_ostream OS(MethodString);
 
-  PrintingPolicy PP = Context.getPrintingPolicy();
-  PP.PolishForDeclaration = true;
-  PP.SuppressStrongLifetime = true;
-  PP.SuppressLifetimeQualifiers = true;
-  PP.SuppressUnwrittenScope = true;
-  for (const ObjCMethodDecl *MD : SelectedMethods) {
+  assert(MethodDeclarations.size() >= SelectedMethods.size() &&
+         "less declarations than selected methods?");
+  for (const auto &I : llvm::enumerate(SelectedMethods)) {
+    indexer::Indexed<const ObjCMethodDecl *> Decl = I.value();
     // Skip methods that are already defined.
```

Comment:
Since Decl.BoolResult could be Unknown, are we also skipping definitions that 'may' already be defined?